.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "thirdparty/real_time_tools"]
 	path = thirdparty/real_time_tools
 	url = git@github.com:KodlabPenn/real_time_tools.git
+[submodule "thirdparty/yaml-cpp"]
```

Comment:
This submodule is not currently locked to a specific branch in `jbeder/yaml-cpp`.  Since this is actively under development, having the latest version may be desirable, but this also introduces some instability.

Reply:
Upon further review, the changes to the `yaml-cpp` `master` branch seem stable enough, and provide useful features.

Reply:
This was actually set to the latest release branch tagged `yaml-cpp-0.7.0`via checking out that commit in particular. We are not on `master` since that is a live branch. We can update and receive those changes when they release again or if we find any bugs that require it.